var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -315,4 +315,12 @@ def nvariant_bool(feature):
         if '%intel' in spec and spec.version >= Version('5.6'):
             cmake_args.append('-DPARAVIEW_ENABLE_MOTIONFX:BOOL=OFF')
 
+        # -no-ipo prevents internal compiler error from multi-file
+        # optimization (https://github.com/spack/spack/issues/18192)
+        if '%intel' in spec:
+            cmake_args.extend([
+                '-DCMAKE_CXX_FLAGS=-no-ipo',
+                '-DCMAKE_C_FLAGS=-no-ipo'
+            ])
```

Comment:
If we have to disable `ipo` can we check which versions of the Intel compiler needs this? Also, another option is to have a conflict for those versions.

Reply:
I need to test the versions I can and make sure, and I would also like to add this to `flag_handler` similar to #18195 . Hope to get this updated today

Reply:
@alalazo I hit this issue with intel@18.0.2 and intel@19.0.4. Also trying with intel@19.1.3, but I have to build everything up from scratch.

I tried with intel@17.0.4, but that failed before getting to paraview. I'm guessing older versions will not make it to paraview either.

Reply:
Unfortunately intel@19.1.3 failed before getting to paraview too.